app/Controllers/Account/AccountController.php

```diff
@@ -86,25 +88,63 @@ public function profile()
             'location'  => ['permit_empty', 'string', 'max_length[255]'],
             'company'   => ['permit_empty', 'string', 'max_length[255]'],
             'signature' => ['permit_empty', 'string', 'max_length[255]'],
+            'avatar'    => ['permit_empty', 'uploaded[avatar]', 'mime_in[avatar,image/jpg,image/jpeg,image/png]', 'max_size[avatar,1024]'],
```

Comment:
Apparently `permit_empty` cannot be used with file validation, because it causes the file validation to be completely ignored.